src/status_im/contexts/profile/login/events.cljs

```diff
@@ -57,14 +57,15 @@
                   [{:method     "wakuext_startMessenger"
                     :on-success [:profile.login/messenger-started]
                     :on-error   #(log/error "failed to start messenger" %)}]]
-                 [:dispatch [:universal-links/generate-profile-url]]
                  [:dispatch [:community/fetch]]
-                 [:push-notifications/load-preferences]
-                 [:profile.config/get-node-config]
+
+                 ;; Wallet initialization can be delayed a little bit because we
+                 ;; need to free the queue for heavier events first, such as
+                 ;; loading chats and communities. This globally helps alleviate
+                 ;; stuttering immediately after login.
+                 [:dispatch-later [{:ms 500 :dispatch [:wallet/initialize]}]]
```

Comment:
It'd be great if we somehow calculate these delays or schedule them for a moment when the app is free.

Otherwise, I think there will be phones with a certain amount of data where this delay is just not enough.

Reply:
@ulisesmac, I agree that would be better, but this is good enough I think compared to `develop` because we currently process dozens of events and signals at login all in the same batch or nearly the same batch, even things that can wait a little. There will be many cases where 500ms won't be enough in real life, but at the same time, we can't delay this part too much, otherwise the wallet won't show up for too long.

I see your suggestion as a further improvement, wdyt?

Reply:
Yes, this comment is more focused in a future improvement and just sharing that this may not be a definitive solution.

ofc, we can investigate it later :+1: